contracts/schemes/ContinuousLockingToken4Reputation.sol

```diff
@@ -144,7 +147,7 @@ contract ContinuousLocking4Reputation is Agreement {
         }
         reputation = uint256(fromReal(reputation));
         require(reputation > 0, "reputation to redeem is 0");
-        // check that the reputation is sum zero
+        // check that the reputation is sum zero [JG: there is no check here]
```

Comment:
this  `reputationRewardLeft = reputationRewardLeft.sub(reputation);` safe math will make sure the contract does not reward more than needed.
we might not need that